torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -195,7 +200,7 @@ RegisterOperators size_eq_guard({
         // if we would ever return refined tensor, which would change aliasing
         // analysis, we should update aliasdb pass.
         [](const Node* node) -> Operation {
-          return [](Stack* stack) {
+          return [](Stack& stack) {
```

Comment:
Did you mean to change this for this PR?  It doesn't seem related.

Reply:
> Minor fixes to remove compiler warning

yes it's intended, the old signature has been deprecated.